Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/WeaponData.xml

```diff
@@ -3071,4 +3071,45 @@
         <AllowedMovement value="Moving"/>
         <TeleportResetRange value="0"/>
     </CWeaponLegacy>
+    <CWeaponLegacy id="AP_VoidRayPurifierBeamLaunch">
+        <EditorCategories value="Race:Protoss"/>
+        <Options index="Hidden" value="1"/>
+        <Icon value="Assets\Textures\btn-upgrade-protoss-airweaponslevel0.dds"/>
+        <DisplayEffect value="AP_VoidRayPurifierBeamDamage"/>
+        <TargetFilters value="Visible,Ground;Missile,Stasis,Dead,Hidden,Invulnerable"/>
+        <Range value="6"/>
+        <RangeSlop value="2"/>
+        <Period value="1.5"/>
+        <Backswing value="0.75"/>
+        <DamagePoint value="0"/>
+        <Effect value="AP_VoidRayPurifierBeamLaunchPersistent"/>
+        <AllowedMovement value="Moving"/>
+        <TeleportResetRange value="0"/>
+        <Options index="OnlyFireWhileInAttackOrder" value="0"/>
+        <Options index="OnlyFireAtAttackOrderTarget" value="0"/>
+        <LegacyOptions index="CanRetargetWhileChanneling" value="1"/>
+    </CWeaponLegacy>
+    <CWeaponStrafe id="AP_VoidRayPurifierBeamDummyWeapon">
+        <Effect value="AP_PointDefenseLaserDummy"/>
+        <EditorCategories value="Race:Protoss"/>
+        <Options index="Hidden" value="1"/>
+        <AcquireCallForHelpFilters value="-;Player,Ally,Neutral,Enemy"/>
+        <TargetFilters value="Visible,Ground;Missile,Stasis,Dead,Hidden,Invulnerable"/>
+        <Range value="1"/>
+<!--        loiter radius = "area of effect" for this weapon-->
+        <LoiterRadius value="2"/>

```

Comment:
Shouldn't match it splash area size (0.5 from `AP_VoidRayPurifierBeamSearch` effect)?

Reply:
Not in this case; the intent is that it's meant to be worse at attacking small individual units, so it's meant to "miss" the target fairly often.